src/features/submitCode/gitPush/ui/GitPushDialog.tsx

```
@@ -42,34 +67,7 @@ export default function GitPushDialog({ githubUrl }: GitPushDialogProps) {
-                        onClick={() => setCurrentRepo(repo.repoName)}
-                        className="text-white hover:primary flex justify-between"
-                      >
-                        <p className="text-secondary text-md"> {repo.repoName}</p>
-                        <p className="text-xs text-[#ccc]">branch : {repo.defaultBranch}</p>
-                      </SelectItem>
-                    ))}
-                  </ul>
-                ) : null}
-              </SelectContent>
-            </Select>
+            <Select option={reposOptions} value={currentRepo} title="" setValue={setCurrentRepo} />
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)